packages/sdks-tests/src/specs/index.ts

```diff
@@ -139,6 +140,7 @@ export const PAGES: Record<string, Page> = {
   '/image-high-priority': { content: imageHighPriority },
   '/image-no-webp': { content: imageNoWebp },
   '/data-bindings': { content: dataBindings },
+  '/nested-symbol': { content: NESTED_SYMBOL_CONTENT },
```

Comment:
IMO, we should change the name of the content to be more accurate. Nested symbols means a symbol inside a symbol (we already have a nested symbols content)
```suggestion
  '/symbols-with-list-content-input': { content: SYMBOLS_WITH_LIST_CONTENT_INPUT },
```
or `/symbols-list-content-input-repeats` etc that more aligns to what you fixed